### ISSUE TITLE
Roadmap: Research: True Intent Survey

### ISSUE THREAD

**Issue by JackieRiley1** [hackforla/website#2663]
### Dependency
- #2733
- Bonnie to setup survey123

### Overview
We need to determine feasibility and priority of the true intent survey so that we know when to work on it.

### Action Items
- [x] Copy [roadmap template](https://github.com/hackforla/website/wiki/Roadmap-template, add it to a comment below, and fill out
- [x] Define requirements for the survey in a spreadsheet 
   - [x] Find commercial options that can achieve some or all of the desired results 
- [ ] See if there are any relevant articles on how to integrate surveys with Google Analytics and note them below
- [ ] Explore lift required with TPM (Technical product lead) and Tech leads as appropriate
   - [ ] Create issue for determine how to add survey to the website 
- [ ] Work with Research Lead to identify where this fits into Research Roadmap

### Resources/Instructions
[Roadmap template](https://github.com/hackforla/website/wiki/Roadmap-template)

[Survey Questions and Instructions](https://docs.google.com/document/d/16b-gVZCT9aI9rdkmEz1GT1Kaj8P5sYtGaT5LKZVtglM/edit?usp=sharing)

[Survey Criteria and Options](https://docs.google.com/spreadsheets/d/1ShPMAC8OiydmPqo5SlD-YwGF1rreED_D13SYpcEcSBA/edit?usp=sharing)

**Comment by R-Tomas-Gonzalez** [comment]
@SAUMILDHANKAR and @JessicaLucindaCheng Hey y'all, this was brought to my attention a while ago. And I don't have the bandwidth for this any longer. I am now gearing up for a new full-time role and I need to put all my attention in studying for that. I assigned myself to this issue today to remind myself to let you guys know about this. Please message [Jackie](https://hackforla.slack.com/team/U02HZPED81X) for more information. If you have any questions you can also message me on slack. 

Thanks,

-Tomas

**Comment by R-Tomas-Gonzalez** [comment]
Also, just wondering... @ExperimentsInHonesty Does this need to go over to design team to have them build out what the survey should look like? That might be helpful in having the lead team get this ball rolling as well.

**Comment by SAUMILDHANKAR** [comment]
Some of the relevant questions that came up during my discussion with Tomas:
True Intent Survey Questions
1. Is this going to live on the website for a certain amount of time?
2. Could this be a reusable component of the website
3. Design - Do we want design to create the visual aspects of the survey?
4.a. Google Forms does not have much flexibility as far as visuals go, can we stray away from Google forms and use something like Mailchimp? 
4.b. Or another third-party product that we can integrate analytics and allows for more visual freedom?
5. Is this a form that we want to build out and style ourselves?
6. From a legal standpoint, if we're utilizing cookies or sessions to keep track of whether or not the user has seen the survey, do we have to disclose that we're storing their cookies?

**Comment by JackieRiley1** [comment]
In response to some of the questions above:

Ideally, we would want at least 400 responses. This may mean leaving it live for a month and then reevaluating. It sounds like there are various ways to create a survey that fits our needs.  I might suggest that the survey's integration into Google analytics be prioritized to allow easier data analysis. We could make minor changes to the survey if that would allow for its integration.

**Comment by ExperimentsInHonesty** [comment]
- This issue originally was too large.  So I moved the creation of the survey to another issue #2733

**Comment by JackieRiley1** [comment]
### Name of initiative
-True Intent Survey

### Problem statement
-We don’t have data to indicate who is using the Hack for LA website, their goals, or if they are successful in achieving their goals.

### Objective/Desired Impact
- Determine the reasons users are visiting the Hack for LA website and if they are successful in achieving their goals.

### Success Metrics

Success may be defined in the following ways: 1) Successful integration of survey and display on website for 1+ month and 2) Adequate response rate*

*Ideally, we would have 100+ responses for each user journey, for a total of 400+ total responses. However, since some groups (e.g., donors,  partners) may have relatively few users represented, this may not be possible, particularly in a short time frame. 

### Types of team members involved
-Product, UX research, possibly design

**Comment by JackieRiley1** [comment]
[Requirements for True Intent Survey](https://docs.google.com/spreadsheets/d/1ShPMAC8OiydmPqo5SlD-YwGF1rreED_D13SYpcEcSBA/edit?usp=sharing)

**Comment by JackieRiley1** [comment]
The True Intent survey has been updated to reflect any changes from previous conversations. The document includes comments based on our discussion from the UX research team. 

I have also created a spreadsheet which states the requirements for the survey and some commercial options. The analytic integrations will depend on the survey selected. Based on the information I have reviewed, if there is a budget for creating the survey, Survey Monkey may be a good option. If there is no budget for the survey, then we would need to rely on a free tool, most likely Google forms. However, Google forms would not allow us to create a pop-up survey; instead it would be a link pasted into the website. Pasting the link would not allow us to intercept the user well and will produce a lower survey response.

**Comment by JackieRiley1** [comment]
Progress: 
The True Intent survey has been updated to reflect any changes from previous conversations. The document includes comments based on our discussion from the UX research team. All questions posed have been answered.

I have also created a spreadsheet which states the requirements for the survey and some commercial options. The analytic integrations will depend on the survey platform selected. Based on the information I have reviewed, if there is a budget for creating the survey, Survey Monkey may be a good option. 

There are products which allow the various survey platforms to iterate with Google analytics, however, they seem to be $100+/month. Based on the purpose of the survey and the cost of the integrations, I think we could do do without the Google analytic integrations for the time being. Lastly, the survey also asks for volunteers to participate in an interview, potentially allowing us to get a better idea of user behaviors without analytics.

Blockers: 
We will need to determine which survey platform to use before we can move forward.

Availability: 
3 hours

ETA: 
This will depend on when the survey platform is decided and if I am able to add the questions myself or someone else will be doing it.

**Comment by sacamp** [comment]
@JackieRiley1 we have made some adjustments to your spreadsheet and have added some potential survey options. Can you go through these options, as well as any other potential survey options you know of, and do a competitor analysis of the features of each, noting them on the spreadsheet?

**Comment by JackieRiley1** [comment]
Progress: I have read the survey tool's information provided on their website, sampled a trial version (when available), and reached out to customer support to complete the table comparing each of the potential survey tools. My findings are summarized in [the True Intent Survey Criteria Spreadsheet](https://docs.google.com/spreadsheets/d/1ShPMAC8OiydmPqo5SlD-YwGF1rreED_D13SYpcEcSBA/edit?usp=sharing)

Blockers: For some of the products, I have not heard back from the agents. While I was able to create a survey using some of the trial versions, I could not always determine how it would be displayed on the website.

Availability: 3 hours

ETA: This will depend on when the survey platform is decided and if I am able to add the questions myself or someone else will be doing it.

**Comment by JackieRiley1** [comment]
Progress: I was informed that I would be able to get a Hack for LA email and information about getting a  free account from Survey123. 

Blockers: Once I can get access to email and a Survey123 account, I can proceed to creating a survey and testing the survey features.

Availability: 3 Hours

ETA: This will depend on when I am able to get access to the email and Survey123 account.

**Comment by ExperimentsInHonesty** [comment]
Bonnie needs to setup survey123
